.npmignore

```diff
@@ -1,11 +1,14 @@
+/.travis/
 /build/
 /examples/
+/generate/
+/guides/
+/lib/
```

Comment:
That makes complete and total sense and I should have realized that, but sometimes I am do the forget.